index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <main></main>
+<title>Page Title</title>
+</head>
+<body>
+
+<h1>My First Heading</h1>
+<p>My first paragraph.</p>
```

Comment:
first paragraph should be in uppercase

Reply:
there was not requirement for it to be uppercase

Reply:
you are right, resolving this thread